first_example.py

```diff
@@ -1,2 +0,0 @@
-remind_me = input('What do you want me to remind you? ')
-print(remind_me)

```

Comment:
remind_me = input('What do you want me to remind you? ')
print("remember: "+ remind_me)